settingtypes.txt

```diff
@@ -0,0 +1,35 @@
+# This file contains settings of minetest_game that can be changed in
+# minetest.conf
+
+#    In creative mode players are able to dig all kind of blocks nearly instantly,
+#    and have access to unlimited resources.
+#    Some of the functionality is only available if this setting is present at startup.
+creative_mode (Creative mode) bool false
+
+#    Flammable nodes will be ignited by nearby igniters. Spreading fire may cause sever harm.
+#    All fire nodes with instantly disappear when fire is disabled.
+enable_fire (Fire) bool true
+
+#    If enabled, steel tools, torches and cobblestone will be given to new players.
+give_initial_stuff (Give initial items) bool false
+
+#    If enabled, players respawn at the bed they last slept at instead of normal spawn.
+#    This setting is only read at startup.
+enable_bed_respawn (Respawn at bed) bool true
+
+#    If enabled, the night can be skipped if more than half of the players are in beds.
+enable_bed_night_skip (Skip night when sleeping) bool true
+
+#    When TNT explodes it destroys nearby nodes, possibl causing a lot of harm on servers.
+#    This setting is disabled by default on servers.
+enable_tnt (TNT) bool true
```

Comment:
I don't understand how this can confuse people. The GUI would display something like "TNT Activated".